hack/bin/helm-template.sh

```diff
@@ -4,25 +4,20 @@
 # hack/helm_vars as overrrides, if available.  This allows debugging helm
 # templating issues without actually installing anything, and without needing
 # access to a kubernetes cluster
-USAGE="Usage: $0"
 
 set -e
 
-chart=${1:?$USAGE}
```

Comment:
what happened to this variable?

Reply:
I don't know how to render a single chart with helmfile.